updated list of unofficial web api (#73)
qbittorrent/wiki: List-of-unofficial-WebAPI-clients.md

- https://github.com/Conan1996-2/qBitorrent-WebUI-API

- https://github.com/taylorcox75/qRemote

    iOS app for managing qBittorrent remotely via its WebAPI, available (free) in App Store

>     VB .NET code to access qBitorrent's WebAPI

Reviewer: ```suggestion

VB .NET code to access qBitorrent's WebAPI

- https://github.com/taylorcox75/qRemote

    iOS app for managing qBittorrent remotely via its WebAPI, available (free) in App Store
```

- https://github.com/Conan1996-2/qBitorrent-WebUI-API

    iOS app for managing qBittorrent remotely via its WebAPI, available (free) in App Store

> - https://github.com/taylorcox75/qRemote

Reviewer: ```suggestion
- https://github.com/taylorcox75/qRemote

    iOS app for managing qBittorrent remotely via its WebAPI, available (free) in App Store
```

- https://github.com/Conan1996-2/qBitorrent-WebUI-API

> VB .NET code to access qBitorrent's WebAPI

Reviewer: ```suggestion
    VB .NET code to access qBitorrent's WebAPI
```

> 
>     VB .NET code to access qBitorrent's WebAPI
> 
> - https://github.com/taylorcox75/qRemote
> 
>     iOS app for managing qBittorrent remotely via its WebAPI, available (free) in App Store
> 

Reviewer: ```suggestion
- https://github.com/taylorcox75/qRemote

    iOS app for managing qBittorrent remotely via its WebAPI, available (free) in App Store
    
- https://github.com/Conan1996-2/qBitorrent-WebUI-API

    VB .NET code to access qBitorrent's WebAPI

```

    VB .NET code to access qBitorrent's WebAPI

- https://github.com/taylorcox75/qRemote

    iOS app for managing qBittorrent remotely via its WebAPI, available (free) in App Store
- https://github.com/taylorcox75/qRemote

    iOS app for managing qBittorrent remotely via its WebAPI, available (free) in App Store
    
- https://github.com/Conan1996-2/qBitorrent-WebUI-API

>     VB .NET code to access qBitorrent's WebAPI

Reviewer: ```suggestion
```
